src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final FigureName figureName = FigureName.ISOSCELES_TRAPEZOID;
```

Comment:
Don't use static fields for the figure name; it should be returned by a method, for example, getName(), which can be declared in the abstract class Figure and overridden here.